From 878403fa8793ce26307bb3f4d14348f99cce047b Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solus-project.com>
Date: Sun, 8 Oct 2017 22:59:32 +0100
Subject: [PATCH] lldb: Fix build error with GCC 7

Original path: Rex Morgan <rex@rexflex.net>

Rebased for LLVM 5

Signed-off-by: Ikey Doherty <ikey@solus-project.com>
---
 source/Utility/TaskPool.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/source/Utility/TaskPool.cpp b/source/Utility/TaskPool.cpp
index d33f23c..3e5518a 100644
--- a/source/Utility/TaskPool.cpp
+++ b/source/Utility/TaskPool.cpp
@@ -12,6 +12,7 @@
 #include <cstdint> // for uint32_t
 #include <queue>   // for queue
 #include <thread>  // for thread
+#include <functional>
 
 namespace {
 class TaskPoolImpl {
-- 
2.14.2

